.travis.yml

```diff
@@ -18,9 +19,9 @@ before_deploy:
   - heroku container:login
```

Comment:
You should be able to remove any mention of Heroku, including any encrypted env var.